src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -360,11 +364,21 @@ private void validateSite(Site site) throws ValidationException {
     }
 
     private void validateApp(App app) throws ValidationException {
-        if (app != null && app.getExt() != null) {
-            try {
-                Json.mapper.treeToValue(app.getExt(), ExtApp.class);
-            } catch (JsonProcessingException e) {
-                throw new ValidationException("request.app.ext object is not valid: %s", e.getMessage());
+        if (app != null) {
+            final String appId = app.getId();
+            if (CollectionUtils.isNotEmpty(blacklistedApps) && StringUtils.isNotBlank(appId)
+                    && blacklistedApps.contains(appId)) {
+                // propagate BlacklistedAccOrApp instantly
+                throw new BlacklistedAccountOrApp(String.format(
```

Comment:
How `BlacklistedAccountOrApp` is catching?
Moreover, i think blacklist checking is not a part of request validation. Maybe it make sense to create separate component for such "security" actions. What do you think?

Reply:
Well, it is NOT being caught all the way up to `handleResult` in `AuctionHandler`. As for "security" component - I don't think it is necessary at this moment, maybe if we get some more related aspects other than blacklisted apps and accounts. 

To me it is more of request-validation related moments, just that if we found either being in the blacklist - we should cancel the request immediately. So maybe it does make sense to check blacklists separately...